modules/http2/h2_h2.c

```diff
@@ -501,7 +475,7 @@ int h2_is_acceptable_connection(conn_rec *c, request_rec *r, int require_all)
 
         /* Check TLS cipher blacklist
          */
-        val = opt_ssl_var_lookup(pool, s, c, NULL, (char*)"SSL_CIPHER");
+        val = ap_ssl_var_lookup(pool, s, c, NULL, (char*)"SSL_CIPHER");
```

Comment:
`ap_ssl_var_lookup` expects `const char *` as last parameter, `var` is `const char *`. What is the purpose of converting to `char *`?